src/status_im/ui/screens/wallet/send/views.cljs

```diff
@@ -148,7 +147,7 @@
 
 (defn toolbar-modal []
   [toolbar/toolbar2 {:style wallet.styles/toolbar}
-   [toolbar/nav-button (act/close-white act/default-handler)]
+   [toolbar/nav-button (act/close-white #(re-frame/dispatch [:wallet/discard-transaction-leave]))]
```

Comment:
we shouldn't discard transaction because this modal screen can be opened from unsigned transactions in wallet

Reply:
so this can be done only for send command transaction